Kitodo/src/main/java/org/kitodo/services/data/TaskService.java

```diff
@@ -660,6 +660,7 @@ public boolean executeScript(Task task, String script, boolean automatic) throws
                 }
             }
         } catch (IOException e) {
+            logger.error(e.getMessage(), e);
```

Comment:
Please replace this log call and following `Helper.setFehlermeldung` with `Helper.setErrorMessage` which bundles both calls.